### PR TITLE
update getting-started docs & example

### LINKED ISSUE #111: [BUG] stale documentation for getting started
**Describe the bug**
Documentation is stale for `getting-started.rst` and `getting-started.ipynb`, specifically where it still uses `impedance.circuits` and `impedance.plotting`. Added a list to the bottom of things I caught, feel free to add

**edit:** There are also some issues with using outdated circuit elements.

**To Reproduce**
See [`getting-started.rst`](https://github.com/ECSHackWeek/impedance.py/blob/master/docs/source/getting-started.rst)

**Expected behavior**
Use the new API:

```python
from impedance.models.circuits import CustomCircuit
```

and

```python
from impedance.visualization import plot_nyquist
```

**Additional context**
i've started a PR to fix this, need to update the notebook still

* old imports in `getting-started.rst`
* `getting-started.ipynb` is stale
* old circuit elements in circuit string in `getting-started.rst`

### PR DESCRIPTION
**Pull request by hkennyv**
* [x] fix imports `getting-started.rst`
* [x] update circuit elements in `getting-started.rst`
* [x] update `getting-started.ipynb`

**updating data imports**
* [x] getting-started
* [x] fitting-example
* [x] model_io_example
* [x] validation_example
* [x] plotting_example

closes #111

### CODE REVIEW

**Comment by hkennyv** [comment]
should be ready to go, could i get a review? @BGerwe @mdmurbach 

btw, how do you normally remove the metadata from notebooks? the `getting-started.ipynb` is short so i just did it by hand but I can see it'd be a total PITA for larger notebooks.

File: docs/source/_static/getting-started.ipynb [hunk 1]
```diff
@@ -15,7 +15,7 @@
    "source": [
     "import numpy as np\n",
     "\n",
-    "data = np.genfromtxt('../data/exampleData.csv', delimiter=',')\n",
+    "data = np.genfromtxt('exampleData.csv', delimiter=',')\n",
```
Comment (BGerwe) [inline docs/source/_static/getting-started.ipynb:18]: Let's modify this block to show off our preprocessing functionality for reading in data and filtering out data points with positive imaginary parts. Most of the other example notebooks use the numpy approach seen here, but I can quickly change those and add a commit to your PR.

**Comment by mdmurbach** [comment]
Cool! Thanks to you both 😊 

For the notebook cleaning, I’ve done something similar to http://timstaley.co.uk/posts/making-git-and-jupyter-notebooks-play-nice/ before for other projects where we really needed good diffs, but I don’t think we’ve ever really worried about it here. I’ve typically just hit the restart kernel and run all as the last step when changing a notebook.

File: docs/source/_static/getting-started.ipynb (hunk 1 above)
Comment (BGerwe) [reply docs/source/_static/getting-started.ipynb:18]: I'm having some trouble adding my commit to this PR. Is the "Allow edits from maintainers." box selected?

Comment (hkennyv) [reply docs/source/_static/getting-started.ipynb:18]: Huh weird @BGerwe, I have it checked on the PR here:
image

Are you committing on the user fork or the ECSHackWeek repository? I believe it needs to be on my user fork

**Comment by hkennyv** [comment]
@BGerwe good call on the review, see my changes.

Does anyone have thoughts on these two different syntaxes? I know it's a silly thing, but IME, sometimes new users may get confused with the tuple destructuring. Is there a preferred way to use this (so that we can keep it consistent in the examples)?

```python
frequencies, Z = readFile('exampleData.csv')
```

vs

```python
data = readFile('exampleData.csv')

frequencies = data[0]
Z = data[1]
```

**Comment by mdmurbach** [comment]
I’d vote for the tuple destructuring over a separate data variable 👍

File: docs/source/getting-started.rst [hunk 2]
```diff
@@ -118,10 +118,9 @@ For this dataset, importing the data looks something like:
...
 
-  data = np.genfromtxt('./exampleData.csv', delimiter=',')
+  from impedance.preprocessing import readFile
 
-  frequencies = data[:,0]
-  Z = data[:,1] + 1j*data[:,2]
+  frequencies, Z = readFile('./exampleData.csv')
 
   # keep only the impedance data in the first quandrant
   frequencies = frequencies[np.imag(Z) < 0]
```
Comment (BGerwe) [inline docs/source/getting-started.rst:126]: We could use `ignoreBelowX` here, but one could also argue this approach is more transparent for beginning users. I'll leave it up to you if you want to change it or not.

File: docs/source/_static/getting-started.ipynb (hunk 1 above)
Comment (BGerwe) [reply docs/source/_static/getting-started.ipynb]: I got it to work this morning, I think I was misspelling something in the push command yesterday... silly. Anyways, this looks great!

File: docs/source/getting-started.rst (hunk 2 above)
Comment (hkennyv) [reply docs/source/getting-started.rst:126]: I think `ignoreBelowX` is sufficiently clear, so let's go with that. Thanks for the suggestion!

**Comment by hkennyv** [comment]
Sweet, I think we are good to go 🚀

**Comment by mdmurbach** [comment]
@all-contributors please add @BGerwe for review

**Pull request merged**